### PR TITLE
Remove spurious __EOF__ hiding existing declarations/tests

### PR DESCRIPTION
**Pull request by MoonlightSentinel**
... and fix broken (public) unittests.

### CODE REVIEW

**Comment by maxhaton** [comment]
```

std/experimental/allocator/package.d(3365:1)[warn]: A unittest should be annotated with at least @safe or @system
--
  | std/experimental/allocator/package.d(3373:54)[warn]: Prefer the new "'alias' identifier '=' type ';'" syntax to the  old "'alias' type identifier ';'" syntax.
  | std/experimental/allocator/package.d(3531:16)[warn]: Local imports should specify the symbols being imported to avoid hiding local symbols.
  | std/experimental/allocator/package.d(3538:16)[warn]: Local imports should specify the symbols being imported to avoid hiding local symbols.
  | std/experimental/allocator/package.d(3569:1)[warn]: A unittest should be annotated with at least @safe or @system
  | std/experimental/allocator/package.d(3714:1)[warn]: A unittest should be annotated with at least @safe or @system
  | std/experimental/allocator/package.d(3752:1)[warn]: A unittest should be annotated with at least @safe or @system
  | std/experimental/allocator/package.d(3821:1)[warn]: A unittest should be annotated with at least @safe or @system
  | std/experimental/allocator/package.d(3846:1)[warn]: A unittest should be annotated with at least @safe or @system


```
Buildkite-style moaning

File: std/experimental/allocator/package.d [hunk 1]
```diff
@@ -3323,8 +3323,6 @@ nothrow:
     theAllocator.dispose(arr);
 }
 
-__EOF__
```
Comment (RazvanN7) [inline std/experimental/allocator/package.d:3326]: LOL. What was the point of it? Was this an accident?

Comment (MoonlightSentinel) [reply std/experimental/allocator/package.d:3326]: Probably, it was introduced at the end of the file just before a `version(none)` declaration.

**Review by RazvanN7 — APPROVED** [review]

File: std/experimental/allocator/package.d (hunk 1 above)
Comment (MoonlightSentinel) [reply std/experimental/allocator/package.d:3326]: https://github.com/dlang/phobos/blob/8b249a6240fe9a923da69c7122fc27059640ff59/std/allocator.d#L3987

Comment (schveiguy) [reply std/experimental/allocator/package.d:3326]: I just was about to create a PR to remove this.

I found it in my attempts to refactor phobos for phobos v2, my refactoring didn't do well with this. @MoonlightSentinel was on the beerconf call when I found it, thanks for fixing it!

> Was this an accident?

I actually emailed Andrei, and he said he remembered some compiler bug making him put something like that in.

**Pull request merged**